app/src/main/java/org/mozilla/fenix/search/SearchDialogController.kt

```diff
@@ -69,6 +70,11 @@ class SearchDialogController(
 ) : SearchController {
 
     override fun handleUrlCommitted(url: String, fromHomeScreen: Boolean) {
+        // Do not load URL if search engine is application added
```

Comment:
Actually, I am wrong about this suggested change. I think instead of `added` this should be `selected`
```suggestion
        // Do not load URL if application search engine is selected.
```

Reply:
I'll update.  Thanks